contrib/pkg/createcluster/create.go

```diff
@@ -383,6 +396,11 @@ func (o *Options) Complete(cmd *cobra.Command, args []string) error {
 		o.HibernateAfterDur = &dur
 	}
 
+	if manualCloud := manualCCOModeClouds[o.Cloud]; manualCloud && !o.CredentialsModeManual {
+		o.CredentialsModeManual = true
+		o.log.Infof("Defaulting credentials mode to Manual for cloud=%s", o.Cloud)
```

Comment:
As above, "Defaulting" is a bit misleading.
```suggestion
		o.log.Infof("Using Manual credentials mode for cloud=%s", o.Cloud)
```